magick-wrapper.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Please add checking this script to `run-tests.sh` so that `shellcheck` would run over it in the CI.